tests/testquic.nim

```diff
@@ -102,8 +102,12 @@ suite "Quic transport":
 
     proc runClient() {.async.} =
       let client = await createTransport(true)
-      let connFut = client.dial("", server.addrs[0])
-      # conn should be refused by server
-      check not await connFut.withTimeout(1.seconds)
+      try:
+        let conn = await client.dial("", server.addrs[0])
+        await conn.close()
```

Comment:
```suggestion
        let conn = await client.dial("", server.addrs[0])
        check false # conn should be refused by client
```

connection should not be established with invalid cert? so if dial is successfully `check false` will ensure to fail test